docs/cymbal-shops.md

```diff
@@ -0,0 +1,25 @@
+# Cymbal Shops Branding
+
+By default, when you deploy this sample app, the "Online Boutique" branding (logo and wording) will be used.
+But you may want to use Google Cloud's fictitious company, _Cymbal Shops_, instead.
+
+To use "Cymbal Shops" branding, set the `CYMBAL_LOGO` environment variable to `"true"` in the the Kubernetes manifest (`.yaml`) for the `frontend` Deployment.
```

Comment:
Was the naming `CYMBAL_LOGO` for this planned? 
``Suggestion
CYMBAL_LOGO -> CYMBAL_BRANDING
``

Reply:
I used `CYMBAL_LOGO` at first to [match Bank of Anthos](https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/release/v0.5.0/kubernetes-manifests/frontend.yaml#L50).
And I 100% agree, `CYMBAL_BRANDING` is a more appropriate name.